import error_notification2

error_notification2.send_finish_message('Finish!')
